metrics/firebase/docs/features/builds_aggregation/diagrams/firestore_create_builds_aggregation_sequence_diagram.puml

```diff
@@ -17,6 +17,12 @@
     FCF -> FCF : triggers an onCreate event handler
     
     FCF -> Firestore : increases a counter inside the 'build_days' collection
+    FCF -> Firestore : increases a successful builds duration inside the 'build_days' collection
```

Comment:
It seems like we should increment the `successfulBuildsduration` only in case when the build status is successful. So, I believe we should show on this diagram that we are incrementing the `counter` inside the document in any case and then explain that we are incrementing the `successfulBuildsduration` in case the created build has the `successful` build status. The same suggestion for the `update builds aggregation` sequence diagram.